packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -32,20 +46,8 @@ const { Z_SYNC_FLUSH } = zlib.constants;
 
 const packageCompassAsync = promisify(packageCompass);
 
-export const COMPASS_PATH = path.dirname(
-  require.resolve('mongodb-compass/package.json')
-);
-export const LOG_PATH = path.resolve(__dirname, '..', '.log');
-const OUTPUT_PATH = path.join(LOG_PATH, 'output');
-export const SCREENSHOTS_PATH = path.join(LOG_PATH, 'screenshots');
-const COVERAGE_PATH = path.join(LOG_PATH, 'coverage');
-
-let MONGODB_VERSION = '';
-let MONGODB_USE_ENTERPRISE =
-  (process.env.MONGODB_VERSION?.endsWith('-enterprise') && 'yes') ?? 'no';
-
 // should we test compass-web (true) or compass electron (false)?
-export const TEST_COMPASS_WEB = process.argv.includes('--test-compass-web');
+export const TEST_COMPASS_WEB = _TEST_COMPASS_WEB;
```

Comment:
I'll do a follow-up to change the exports for all re-exported vars, just didn't want to make this patch too noisy